guides/common/modules/con_defining-your-content-library.adoc

```diff
@@ -32,8 +40,8 @@ You can synchronize repositories manually, or you can create a sync plan to ensu
 
 .Additional resources
 ifdef::katello[]
-* For more information, see {ContentManagementDocURL}Basic_Content_Management_Workflow_content-management[Basic content management workflow] and {ContentManagementDocURL}Importing_Custom_Content[Importing custom content] in _{ContentManagementDocTitle}_.
-endif::[]
-ifndef::katello[]
-* For more information, including procedures for enabling and synchronizing repositories, see {ContentManagementDocURL}Importing_Custom_Content[Importing custom content] in _{ContentManagementDocTitle}_.
+* For more information about basic workflows, see {ContentManagementDocURL}Basic_Content_Management_Workflow_content-management[Basic content management workflow].
+* For more information about Deb content, see {ContentManagementDocURL}Adding_Custom_Deb_Repositories_content-management[Adding Deb repositories] in _{ContentManagementDocTitle}_.
+* For more information about Yum content, see {ContentManagementDocURL}Adding_Custom_RPM_Repositories_content-management[Adding RPM repositories] in _{ContentManagementDocTitle}_.
```

Comment:
I prefer the original (shorter) list of references because I don't think that a link whose label says "Basic workflow" needs to be introduced with the words "basic workflows" :) And it's similar for the other links. But that's just FYI to explain why I wrote the original list of references the way it was before. This is your PR and your call, obviously :)

Reply:
I have removed "basic workflow" from the description.